platformio.ini

```diff
@@ -12,3 +12,9 @@
 platform = ststm32
 board = nucleo_l432kc
 framework = arduino
```

Comment:
You'll want to add these lines once we add in CAN and the `embedded-pio` submodule:
```
build_flags = -DHAL_CAN_MODULE_ENABLED
lib_extra_dirs = ./embedded-pio
```